docs/source/contributing.rst

```diff
@@ -105,6 +105,7 @@ ideas, code, and documentation to the cf library:
 
 * A James Phillips
 * Alan Iwi
+* Ankit Bhandekar
```

Comment:
Welcome to the club! It's a shame you weren't quite first in the alphabetised list, but I imagine you will cope...

Reply:
Thank you Sadie! :relieved: